webots_ros2_driver/webots_ros2_driver/webots_launcher.py

```diff
@@ -54,7 +54,7 @@ def perform(self, context):
 
 class WebotsLauncher(ExecuteProcess):
     def __init__(self, output='screen', world=None, gui=True, mode='realtime', stream=False, ros2_supervisor=False,
-                 port='1234', **kwargs):
+                 port='1234', host_ip=None, host_port=2000, **kwargs):
         if sys.platform == 'win32':
```

Comment:
It would be nice to add a comment to explain the different between `port` and `host_port` as it may not be obvious to all.

Reply:
Either a comment or maybe renaming them to something like webots_port and simulation_server_port

Reply:
Yes, a renaming would be even better.